sqlstore/src/main/resources/db/migration/V8200.sql

```diff
@@ -0,0 +1,28 @@
+-- serviceboards
+
+-- Create table for instrument ServiceRecords
+CREATE TABLE Instrument_ServiceRecord (
+  recordId bigint(20) NOT NULL AUTO_INCREMENT,
```

Comment:
This should reference the `ServiceRecord (recordId)` field directly, with foreign key constraint, and the table's primary key should be on both ID columns.